tests/ptf/ptf_runner.py

```diff
@@ -92,6 +94,19 @@ def update_config(p4info_path, bmv2_json_path, tofino_bin_path,
     channel = grpc.insecure_channel(grpc_addr)
     stub = p4runtime_pb2_grpc.P4RuntimeStub(channel)
 
+    # Create new target proto object for testvectors
+    if bmv2_json_path is not None:
+        tv_target = targetutils.get_new_target(grpc_addr, target_id="bmv2")
+    else:
+        tv_target = targetutils.get_new_target(grpc_addr, target_id="tofino")
+    # Write the target proto object to testvectors/target.pb.txt
+    tv_base_dir = os.getcwd() + "/testvectors/"
```

Comment:
I've seen this pattern in many places. We set a directory (we hardcode `/testvectors` many times which is not great), create that if not exists, and write a file into it. Perhaps this is something that should go into tvutils?

Reply:
I've been thinking about hardcoding in multiple places too. I can create a common utils in testvectors for tv, portmap, target that takes a proto object and absolute filename as arguments and writes to it.